simpleclient_dropwizard/pom.xml

```diff
@@ -42,12 +42,29 @@
             <artifactId>metrics-core</artifactId>
             <version>3.1.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
```

Comment:
Avoid adding dependencies, we don't want to clash with other deps the user has,

Reply:
Done, that dependency was overkill